CHANGELOG.rdoc

```diff
@@ -2,6 +2,7 @@
 
 Bugs:
 * #978 - Fix for invalid chars being left in a string for invalid b_value from encoding (kjg)
+* #998 - ParameterHash fixes to allow for blanks and to properly transcode the values (kjg)
```

Comment:
Users don't know what `ParameterHash` is or how this is relevant to their apps. What messages can now be parsed that couldn't before?